Assets/Scripts/Game/UserInterface/HUDCrosshair.cs

```diff
@@ -47,12 +46,7 @@ public override void Update()
 
         void LoadAssets()
         {
-            if (TextureReplacement.CustomTextureExist(defaultCrosshairFilename))
-                CrosshairTexture = TextureReplacement.LoadCustomTexture(defaultCrosshairFilename);
-            else
-                CrosshairTexture = Resources.Load<Texture2D>(defaultCrosshairFilename);
-
-            crosshairSize = TextureReplacement.GetSize(CrosshairTexture, defaultCrosshairFilename, true);
+            CrosshairTexture = DaggerfallUI.GetTextureFromResources(defaultCrosshairFilename, out crosshairSize);
```

Comment:
I just copied this last week for loading images for the icon UI component. Should I update it to use this method then?

Reply:
Hi, yes. GetSize() is now only intended for Img and CifRci (gets original texture size for panel layout if xml is not found), GetTextureFromResources is a simple helper method which set size as resolution if there is no xml. The biggest difference is that now it also loads textures from .dfmods